packages/console/src/pages/ApplicationDetails/components/AddAnotherButton/index.module.scss

```diff
@@ -0,0 +1,3 @@
+.addButton {
+  cursor: pointer;
```

Comment:
nit: should we have the cursor set for all IconButton Component, could put it in some common shared button styles module.

Reply:
we have LOG-1890

Reply:
rebased